### ISSUE TITLE
Focus indicator is missing from a grid cell when clicking once

### ISSUE THREAD

**Issue by johannesh2** [vaadin/vaadin-grid-pro#61]
I click a cell once and I expect to see focus ring around the cell so I know that typing will occur in that particular cell. Double-click behaves nicely.

**Comment by web-padawan** [comment]
Should the focus ring only apply to edit column, or the whole grid including non-editable column?
